views/recipes.blade.php

```diff
@@ -174,7 +174,7 @@
 					@if(GROCY_FEATURE_FLAG_STOCK_PRICE_TRACKING)
 					<div class="col-6">
 						<label>{{ $__t('Costs') }}&nbsp;&nbsp;
-							<span class="small text-muted">{{ $__t('Based on the prices of the last purchase per product') }}</span>
+							<span class="small text-muted">{{ $__t('Based on the prices of the default consume rule which is "First expiring first, then first in first out"') }}</span>
```

Comment:
It may be more UI friendly if we removed this from the small text and just added it as hover text?

Reply:
Yes, also thought about that for all places where we have that currently, maybe a little help/info icon where the text will be shown on hover...

But fits better in your other pull request #776, as this is more about the logic behind recipe costs.

Reply:
Thanks for the suggestion. Hover text is not used anywhere else at the moment that I am aware of so I would not want to start with just this. Hints (small text) are used throughout the UI.

Reply:
Agree. Will look to add it in my PR. @kriddles Only one way to start 😉